From: Iceyer <iceyers@gmail.com>
Date: Fri, 4 Aug 2017 19:03:59 +0800
Subject: Fix make check failed
Forwarded: https://code.launchpad.net/~iceyers/gsettings-qt/fix-link-path

This commit would fix the failure when invoking "make check".
---
 tests/cpptest.pro | 3 ++-
 tests/tests.pro   | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/cpptest.pro tests/cpptest.pro
index 68b3b49..ec46730 100644
--- a/tests/cpptest.pro
+++ tests/cpptest.pro
@@ -3,7 +3,8 @@ QT += testlib
 QT -= gui
 CONFIG += testcase link_pkgconfig
 TARGET = cpptest
-IMPORTPATH = $$PWD/..
+IMPORTPATH = $$(PWD)/..
+QMAKE_RPATHDIR += $$(PWD)/../src
 SOURCES = cpptest.cpp
 INCLUDEPATH += $$(PWD)/../src
 LIBS += -L$$(PWD)/../src -lgsettings-qt
diff --git a/tests/tests.pro tests/tests.pro
index 6e3a388..2183584 100644
--- a/tests/tests.pro
+++ tests/tests.pro
@@ -4,6 +4,7 @@ QT -= gui
 CONFIG += qmltestcase
 TARGET = test
 IMPORTPATH = $$PWD/..
+QMAKE_RPATHDIR += $$PWD/../src
 SOURCES = test.cpp
 
 schema.target = gschemas.compiled
